src/store/reducers/createMapReducer/openGroupsReducer.js

```diff
@@ -3,13 +3,20 @@ import defaultState from '../../defaultState';
 
 function toggleGroups(groupState, action) {
```

Comment:
It seems that `toggleCategories` in the openCategoriesReducer module and `toggleGroups` here (and possibly others) are similar.  Is it possible to combine them into one in pursuit of DRYness?

Reply:
Yea sure. I pushed the change